server/src/test/java/io/druid/client/CachingClusteredClientFunctionalityTest.java

```diff
@@ -166,7 +166,7 @@ public QueryableDruidServer pick(TreeMap<Integer, Set<QueryableDruidServer>> pri
 
               @Override
               public List<QueryableDruidServer> pick(
-                  TreeMap<Integer, Set<QueryableDruidServer>> prioritizedServers,
+                  SortedMap<Integer, Set<QueryableDruidServer>> prioritizedServers,
                   DataSegment segment,
                   int numServerToPick
```

Comment:
Please change other occurrences of "numServerToPick" to "numServersToPick" in the codebase